packages/stripe-integration/src/stripe-upe/stripe-upe-script-loader.ts

```diff
@@ -2,6 +2,8 @@ import { ScriptLoader } from '@bigcommerce/script-loader';
 
 import { PaymentMethodClientUnavailableError } from '@bigcommerce/checkout-sdk/payment-integration-api';
 
+import { StripeConfigurationOptions } from '../stripev3/stripev3';
```

Comment:
we should not use Stripe V3 interfaces for Stripe UPE, please copy it to UPE with necessary options.